webpack/scenes/ContentViews/Details/Filters/CVModuleStreamFilterContent.js

```diff
@@ -194,8 +194,8 @@ const CVModuleStreamFilterContent = ({
     },
   ];
 
-  const emptyContentTitle = __('No rules have been added to this filter.');
-  const emptyContentBody = __("Add to this filter using the 'Add filter rule' button.");
+  const emptyContentTitle = __('No module stream filter rules yet');
+  const emptyContentBody = __('Add repositories to content view with module streams to select them here.');
```

Comment:
Same here.. Is the message about not having module stream filter rules? Or not having module streams in the content view?

Reply:
Shouldn't emptyContentTitle and emptyContentBody match, then?